src/commands/mod.rs

```diff
@@ -623,7 +635,7 @@ struct RestoreArgs {
 /// changes into or out of the parent revision.
 #[derive(clap::Args, Clone, Debug)]
 struct DiffeditArgs {
-    /// The revision to touch up. Defaults to @ if --to/--from are not
+    /// The revision to touch up. Defaults to @ if neither --to nor --from are
```

Comment:
Doesn't it default to `@` even if `--from` is used?

Maybe there should be a place where we could put examples of the various combinations:

- `jj restore PATHS`: restore `PATHS` from `@-` to `@`
- `jj restore -r REV PATHS`: restore `PATHS` from `REV-` to `REV`
- `jj restore --from FROMREV PATHS`: restore `PATHS` from `FROMREV` to `@`
- `jj restore --to TOREV PATHS`: restore `PATHS` from `@` to `TOREV`
- `jj restore --from FROMREV --to TOREV PATHS`: restore `PATHS` from `FROMREV` to `TOREV`

If `PATHS` is absent, restore all files. Empty commits will not be abandonned.

Reply:
I haven't figured out a way to improve this. It *is* explained a little better in the docstring fo `DiffeditArgs` itself (the preamble of the help text).